src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -731,6 +733,103 @@ public void SendPayload_ListIsOrdered()
             telemetryRepositoryMock.Verify(x => x.Save(), Times.Once);
         }
 
+        [TestMethod]
+        public void QuickFixApplied_NoPreviousIssues_TelemetryUpdated()
```

Comment:
This is very blatant copy-paste of the secrets tests. The product code and the tests are identical except the list is different, but I can't think of a neater way to do this.